src/content/releases/_index.en.md

```diff
@@ -1,10 +1,27 @@
 +++
-date = 2016-04-09T16:50:16+02:00
+date = 2020-05-04T16:50:16+02:00
 title = "Releases"
 pre = "<b>4. </b>"
 weight = 15
 +++
 
+## v0.3.0 Lighthouse service discover without kubefed
+
+> This release is focused on removing the kubefed dependency from Lighthouse, the user experience
+> and adding experimental wireguard support
```

Comment:
experimental support for wireguard as an alternative to IPsec?

Reply:
```suggestion
> and adding experimental wireguard support as an alternative to IPsec.
```

Reply:
```suggestion
> and adding experimental WireGuard support as an alternative to IPsec
```